config/systems-uenv/santis-uenv.py

```diff
@@ -51,40 +51,23 @@
 
 partitions = [
     {
-        'name': 'mc',
         'name': 'normal',
         'scheduler': 'slurm',
         'time_limit': '10m',
         'environs': environ_names,
-        'container_platforms': [
-            {
-                'type': 'Sarus',
-            },
-            {
-                'type': 'Singularity',
-            }
-        ],
         'max_jobs': 100,
-        'extras': {
-            'cn_memory': 500,
```

Comment:
can we keep this ?

Reply:
see https://github.com/eth-cscs/cscs-reframe-tests/blob/alps/checks/system/slurm/slurm.py#L267